src/plotting/plotting.py

```diff
@@ -277,9 +277,11 @@ def style_plot(fig, axes):
         ax.set_xlabel("")
         ax.grid(axis="y")
         ax = format_date_axis(ax)
-        ax.get_yaxis().set_major_formatter(
-            ticker.FuncFormatter(lambda x, p: format(int(x), ","))
-        )
+        # format thousands as 3,456 if the y values are that large
+        if ax.yaxis.get_majorticklocs().max() > 1000:
```

Comment:
Why do you need the if condition? I thought numbers smaller than 1,000 are automatically handled and have no comma.

Reply:
They do but they are converted to int. Then the y axes for all our multipliers and shares only show 0 and 1 instead of 0.2, 0.4, etc.